pwndbg/aglib/arch.py

```diff
@@ -1,8 +1,18 @@
 from __future__ import annotations
 
+from typing import Literal
+
+import pwnlib
+
 import pwndbg
 from pwndbg.lib.arch import Arch
 
+# We will optimize this module in the future, by having it work in the same
+# way the `gdblib` version of it works, and that will come at the same
+# time this module gets expanded to have the full feature set of its `gdlib`
```

Comment:
```suggestion
# time this module gets expanded to have the full feature set of its `gdblib`
```

Reply:
Actually, it already works like the `gdblib` version. That comment is outdated. I'll just take it out.